website/public/locales/en/message.json

```diff
@@ -2,6 +2,7 @@
   "confirm_open_link_body": "Do you want to open this link?",
   "confirm_open_link_header": "Confirm open link",
   "copy_message_id": "Copy message ID",
+  "copy_message_text": "Copy message text",
```

Comment:
Maybe something like "Copy raw message" is better here?

Reply:
That also works. I used "Copy message text" to keep it more in line with "Copy message ID", but raw text may be more accurate. will update

Reply:
changed